packages/shared/src/components/modals/RecommendAnArticle.tsx

```diff
@@ -0,0 +1,182 @@
+        <h3 className="font-bold typo-title3">Recommend an article</h3>
+        <ModalCloseButton onClick={onRequestClose} />
+      </header>
+      <section className="overflow-auto relative px-10 pt-6 pb-10 w-full h-full shrink max-h-full">
+        <p className="mb-2 typo-callout text-theme-label-tertiary">
+          Found an interesting article? Do you want to share it with the
+          community? Enter the article&apos;s URL / link below to add it to the
+          feed.
+        </p>
+        <a
+          className="font-bold underline typo-callout text-theme-label-link"
+          target="_blank"
+          rel="noopener"
+          href={contentGuidlines}
+        >
+          Content guidelines
+        </a>
+        <p className="mt-6 mb-4 typo-callout">Daily suggestions used 0/3</p>
+        <form
+          className="w-full"
+          ref={recommendFormRef}
+          aria-busy={isValidating}
+          onSubmit={onSubmit}
+        >
+          <TextField
+            autoFocus
+            type="url"
+            leftIcon={<LinkIcon />}
+            fieldType="tertiary"
+            name="articleUrl"
+            inputId="article_url"
+            label="Paste article url"
+            disabled={!isEnabled}
+            hint={urlHint}
+            valid={!urlHint}
+            valueChanged={onUrlChanged}
+          />
+          {isSubmitted && (
+            <>
+              <div className="flex items-center mt-4 mb-6 h-8 font-bold rounded-10 w-fit bg-theme-status-success text-theme-label-invert padding px-[0.9375rem]">
+                Request sent
+              </div>
+              <p className="mb-2 typo-callout text-theme-label-tertiary">
+                You will be notified via email about the article request status.
+              </p>
+            </>
+          )}
+
+          {isEnabled ? (
+            <Button
+              className="mt-4 btn-primary "
+              buttonSize="small"
+              type="submit"
+              aria-label="Submit article"
+              disabled={!isEnabled || !enableSubmission}
+            >
+              <span className={isValidating && 'invisible'}>
+                Submit article
+              </span>
+              {isValidating && (
+                <Loader className="absolute inset-x-0 top-1 mx-auto" />
+              )}
+            </Button>
+          ) : (
+            <Button
+              tag="a"
+              href={requestAccessRecommendAnArticle}
+              target="_blank"
+              rel="noopener"
+              aria-label="Request access"
+              className="mt-4 w-fit btn-secondary"
+            >
+              Request access
+            </Button>
+          )}
+        </form>
+      </section>
+      {existingArticle && (
+        <div>
+          <div className="px-10 pb-3.5 border-b border-theme-divider-tertiary">
+            <p className="font-bold typo-callout">Article exists</p>
+          </div>
```

Comment:
Not sure if my suggestion of removing the container is actually a good idea, though I think `h4` would probably be more fitting semantically. What do you think? Also, just wanted to check if the additional empty `div` can be removed or not?
```suggestion
        <div>
          <h4 className="px-10 pb-3.5 border-b border-theme-divider-tertiary font-bold typo-callout">Article exists</h4>
```

Reply:
Good point, I'll look if I can remove the extra one, and agree on the h4 semanticly seen.